File: java/bypstest-api/src/com/wilutions/byps/test/api/inherit/FruitService.java
package com.wilutions.byps.test.api.inherit;

import com.wilutions.byps.RemoteException;

/**
 * @BSessionParamType com.wilutions.byps.test.api.auth.SessionInfo
 *
 *
 */
public interface FruitService extends PlantService {

  public String squeeze() throws RemoteException;
  
  
}

File: csharp/bypstest-ser/src-ser/byps/test/api/inherit/FruitService.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.inherit
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface FruitService : BRemote, 
		PlantService {
	
		String Squeeze();
		void Squeeze(BAsyncResult<String> asyncResult) ;
		Task<String> SqueezeAsync();
		
		
	}
}  // end namespace
